nikola/image_processing.py

```diff
@@ -188,8 +188,8 @@ def resize_svg(self, src, dst, max_size, bigger_panoramas):
                 op = gzip.GzipFile(dst, 'wb')
             else:
                 op = open(dst, 'wb')
-            op.write(lxml.etree.tostring(tree))
-            op.close()
+            with op:
```

Comment:
I’d rather not, `.close()` is more obvious.

Reply:
OK, but if there is an error in `lxml.etree.tostring(tree)` the file would not be closed. But still, I revert this change.